From e66ec7b7e80db1230d98fe5e5ca931ab0936f054 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Fri, 12 Aug 2022 13:44:26 +0800
Subject: [PATCH] Revert "Linux 5.10.136"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 6eae1503ddf94b4c3581092d566b17ed12d80f20.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 1730698124c7..5f4dbcb43307 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 136
+SUBLEVEL = 135
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.34.1

